improve JS example (#13)
UltimateTournament/ArcadeThirdPartyDocs: docs/supported-game-types/real-time.md

# Real-time Multiplayer

> While in async-multiplayer games you interact with your opponents only indirectly (e.g. competing for a better score) in real-time multiplayer games directly interact with them.

Reviewer: ```suggestion
While in async-multiplayer games you interact with your opponents only indirectly (e.g. competing for a better score) in real-time multiplayer games players directly interact with each other.
```

In this category we need to differentiate between two kinds of games:
**Latency sensitive** (fast paced action games) and **not latency sensitive** games (mostly round-based games).

While latency sensitive games are possible (like [cyberworms](https://ultimatearcade.io/cyber-worms) and [cyberworms battle royale](https://ultimatearcade.io/cyber-worms?mode=battle-royale)), they are far more difficult to build and maintain.

If your game doesn't mind if there's a few hundred milliseconds delay, then it's far easier to host them and keep up a great player experience.

> For real-time games we support to types of games

Reviewer: ```suggestion
For real-time games we support two types of games:
```

> * *Player vs Player (PvP)* - players come and go and might fight each other in an open session

Reviewer: ```suggestion
* *Player vs Player (PvP)* - players come and go, might fight each other in an open session
```
